### DISCUSSION TITLE
Disable timestamps for the transcripts

### DISCUSSION THREAD

**Discussion by marekhac** [ggml-org/whisper.cpp#1596] [Ideas]
Guys, please enlight me. Is there a way to disable timestamp for the transcriptions made by Whisper ?

**Comment by bobqianic** [comment]
> Is there a way to disable timestamp for the transcriptions made by Whisper ?

Yes. Using the `-nt` flag will disable the timestamp.

**Reply by marekhac** [reply]
Great! Thank you very much!

**Comment by fleilanio** [comment]
Set output format to txt: `--output_format txt`
